srcpkgs/exult/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'exult'
+pkgname=exult
+version=1.12.0
+revision=1
+archs="i686* x86_64*"
```

Comment:
Is this actually limited to those architectures or just a cross compilation issue?

Reply:
No, I just only have tested x86_64 (and even then only -musl).
Looks like the CI managed to compile on other arches, so maybe I can remove this arbitrary arch limitation...

Reply:
DONE, reported upstream. Followed their advices, disabled tools & i686 build for now, kept the nocross as-is.